src/pudl/transform/eia860.py

```diff
@@ -410,6 +410,52 @@ def _core_eia860__generators_solar(
     return solar_df
 
 
+@asset
+def _core_eia860__generators_energy_storage(
+    raw_eia860__generator_energy_storage_existing: pd.DataFrame,
+    raw_eia860__generator_energy_storage_retired: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the energy storage specific generators table.
+
+    Notes for possible future cleaning:
+    * A lot of the values that are repeated in the generators table are dropped post
+    harvesting. It would be nice to compare the values before dropping them to make sure
+    these specific generator tables don't contain information that the big generator
+    tables does not.
```

Comment:
the harvested columns are integrated into the generators/plants table which is subtly different than being dropped. They don't end up in the storage table, but that is because they are slurped up and associated with that generator. If there was substantial difference between in the values in this table and the other tables, there would be a failure in the harvesting step.